src/wmtk/function/utils/SimplexGetter.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <wmtk/Tuple.hpp>
+#include <wmtk/attribute/ConstAccessor.hpp>
+
+namespace wmtk::function::utils {
+
+template <typename T>
+std::tuple<std::vector<Eigen::Matrix<T, Eigen::Dynamic, 1>>, size_t> get_simplex_attributes(
```

Comment:
I'd like some documentation on that one. I have no clue what the `vertex_marker` does for example.